tv/pages/rooms/[id].tsx

```diff
@@ -0,0 +1,44 @@
+import { getSessions } from '../../api/queries/getSessions';
+import { getRooms } from '../../api/queries/getRooms';
+import { Session } from '../../interfaces';
+import RoomDisplay from '../../components/RoomDisplay';
+
+type RoomProps = {
+  sessions: Session[];
+  preview: boolean;
```

Comment:
Is `preview` used? Could it be removed?

Reply:
All "preview" removed. It could be used for Draft mode, but removed for now as it is not clear whether we ever get to it.